src/Integration.UnitTests/Suppression/SuppressedIssueMatcherTests.cs

```diff
@@ -22,6 +22,7 @@
 using FluentAssertions;
```

Comment:
For all the existing tests I made the server side issue be suppressed=true simply so it doesnt affect what the tests were doing. Then I added a new test that only checks the isSuppressed state.